src/main/java/gitflow/actions/RepoActions.java

```diff
@@ -85,13 +84,13 @@
                 actionList.add(new FinishFeatureAction(myRepo));
 
                 //can't publish feature if it's already published
-                if (branchUtil.isCurrentBranchPublished()==false){
+                if (!branchUtil.isCurrentBranchPublished()){
```

Comment:
Please retain the existing conditional formatting. It's more verbose for readability purposes